pkgs/top-level/perl-packages.nix

```
@@ -21202,6 +21202,21 @@ let
     };
   };
 
+  XMLRPCLite = buildPerlPackage {
+    pname = "XMLRPC-Lite";
+    version = "0.717";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/P/PH/PHRED/XMLRPC-Lite-0.717.tar.gz";
+      sha256 = "0925md6jhzgpsibwgny4my461b2wngm8dhxlcry8pbqzrgrab7rs";
+    };
+    doCheck = true;
+    checkInputs = [ SOAPLite ];
+    meta = {
+      inherit (perl.meta) license;
```

```suggestion
      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
```
It's good to follow the convention of the resst of the file, imho.

---

alright